examples/data-sources/git_repository/data_source_cluster_group.tf

```diff
@@ -0,0 +1,12 @@
+# Read Tanzu Mission Control git repository : fetch cluster group git repository details
+resource "tanzu-mission-control_git_repository" "read_cluster_group_git_repository" {
+  name = "tf-git-repository-name" # Required
+
+  namespace_name = "tf-namespace" #Required
+
+  scope {
+    cluster_group {
+      name = "default" # Required
```

Comment:
Is it required field or has an optional value?

Reply:
it's required field